app/routes/check-details.js

```diff
@@ -59,10 +61,18 @@ export const checkDetailsHandlers = [
         setEndemicsClaim(request, confirmCheckDetailsKey, confirmCheckDetails);
 
         if (confirmCheckDetails === "yes") {
+          const redirectToApply = getSignInRedirect(request, sessionKeys.signInRedirect);
+
+          if (redirectToApply === true) {
+            return h.redirect(`${applyServiceUri}/endemics/you-can-claim-multiple`);
+          }
+
           return h.redirect("/vet-visits");
         }
 
-        return h.view("update-details", { ruralPaymentsAgency: RPA_CONTACT_DETAILS });
+        const { organisation } = getEndemicsClaim(request);
+
+        return h.view("update-details", { devMode: config.devLogin.enabled, sfdButtonLink: `/sign-in?relationshipId=${organisation.id}`});
```

Comment:
As we're getting ready for the SFD integration, might be worth pulling out the sfd URL to a configurable environment variable / service level variable now, ready for them giving us that.

Reply:
Yeah if thats ok!